include/particles/particle.h

```diff
@@ -307,12 +309,12 @@ class Particle : public ParticleBase<Tdim> {
   using ParticleBase<Tdim>::state_variables_;
   //! Neighbour particles
   using ParticleBase<Tdim>::neighbours_;
-  //! Volumetric mass density (mass / volume)
-  double mass_density_{0.};
-  //! Mass
-  double mass_{0.};
-  //! Volume
-  double volume_{0.};
```

Comment:
I thought we were going to keep volume inside particle, no?

Reply:
I thought there might be a chance to have `liquid_volume` too in the future, so thinking to have them inside `properties`.